### ISSUE TITLE
Trying to convert the models from https://github.com/chargebee/chargebee-go

### ISSUE THREAD

**Issue by morrislaptop** [tkrajina/typescriptify-golang-structs#17]
Having some trouble? panic: cannot find type for ptr (addons/)

**Comment by morrislaptop** [comment]
Here you go...

```go
package main

import "github.com/tkrajina/typescriptify-golang-structs/typescriptify"
import (
	// "../chargebee-go/models/subscription"
	"github.com/chargebee/chargebee-go/models/subscription"
)

func main() {
	converter := typescriptify.New()
	converter.CreateFromMethod = false
	converter.Indent = "    "
	converter.BackupDir = ""

	converter.Add(subscription.CreateForCustomerRequestParams{})

	err := converter.ConvertToFile("example/example.ts")
	if err != nil {
		panic(err.Error())
	}
}
```

**Comment by tkrajina** [comment]
Thanks!

OK, so, the problem here is that this project doesn't work with arrays to pointers. I added this feature in the `array-ptrs` branch. But I need to fix a few test before merging it to master.

There is another problem. Your models also contain maps. There is an experimental support for maps in the `maps` branch. You can also use maps if you specify a `ts_type` tag directly in your models (see the example in `README.md`) but since your models are in another project, this could be a problem.

**Comment by tkrajina** [comment]
Duplicate of #12 and will be fixed when #18 is merged.
